bin/node-pg-migrate

```diff
@@ -306,27 +311,21 @@ function readJson(json) {
   }
 }
 
-try {
-  // Load config (and suppress the no-config-warning)
-  const oldSuppressWarning = process.env.SUPPRESS_NO_CONFIG_WARNING
-  process.env.SUPPRESS_NO_CONFIG_WARNING = 1
-  const config = require('config') // eslint-disable-line global-require,import/no-extraneous-dependencies
-  if (config.has(argv[configValueArg])) {
-    const db = config.get(argv[configValueArg])
-    readJson(db)
-  }
-  process.env.SUPPRESS_NO_CONFIG_WARNING = oldSuppressWarning
-} catch (err) {
-  if (err.code !== 'MODULE_NOT_FOUND') {
-    throw err
-  }
+// Load config (and suppress the no-config-warning)
+const oldSuppressWarning = process.env.SUPPRESS_NO_CONFIG_WARNING
+process.env.SUPPRESS_NO_CONFIG_WARNING = 1
+const config = tryRequire('config')
+if (config && config.has(argv[configValueArg])) {
+  const db = config.get(argv[configValueArg])
+  readJson(db)
 }
+process.env.SUPPRESS_NO_CONFIG_WARNING = oldSuppressWarning
 
 const configFileName = argv[configFileArg]
 if (configFileName) {
   // eslint-disable-next-line global-require,import/no-dynamic-require,security/detect-non-literal-require
-  const config = require(path.resolve(configFileName))
-  readJson(config)
+  const jsonConfig = require(path.resolve(configFileName))
+  readJson(jsonConfig)
```

Comment:
I had to change the name as it conflicted with another variable that just became global when I dropped the try/catch.